app/serializers/api/participant_serializer.rb

```diff
@@ -0,0 +1,103 @@
+module API
+  class ParticipantSerializer < Blueprinter::Base
+    identifier :ecf_id, name: :id
+    field(:type) { "npq-participant" }
+
+    class AttributesSerializer < Blueprinter::Base
```

Comment:
could you add the `participant_id` please?

Reply:
also could we ensure the order is similar to ecf please if possible with timestamps at the end?